src/resources/usage/visitusage.txt

```diff
@@ -45,7 +45,9 @@
 
         -window_anchor <x,y> The x,y position on the screen where VisIt's GUI
                              will show its windows (Main window excluded).
-        -style <style>       One of: windows,cde,motif,sgi.
+        -style <style>       One of: Windows,Fusion. If more options are
```

Comment:
I understand the desire NOT to have to encode *here* something that matches with stuff that may optionally appear somewhere else. But, am just curious about spelling and casing here. Not so much about *documenting* that behavior but if I see `FooBar` in Options->Appearance, do I write `-style foobar` here or `-style FooBar`...does it matter? IMHO, best of we make it case-insensitive. Again, only if it really matters.

Reply:
I checked the gui and the styles were presented with beginning capital letters, so I used that. It might matter because the style is set in Qt. Of course we could code around that if we wanted.